<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FitPlaneForm</class>
 <widget class="QWidget" name="FitPlaneForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1509</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>38</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_points_cloud">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>9</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开点云</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>38</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ip地址</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>779</x>
     <y>153</y>
     <width>258</width>
     <height>194</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2"/>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>64</y>
     <width>77</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="connect_camera">
      <property name="text">
       <string>连接相机</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="disconnect_camera">
      <property name="text">
       <string>断开相机</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="start_get_image">
      <property name="text">
       <string>Gocator拍照</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>230</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
